src/domain/Api/index.js

```diff
@@ -42,6 +55,24 @@ export const fetch = (url, options = {}) => {
   });
 };
 
+function getDefaultFetchOpts(options) {
+  const isCorsSimpleMethod =
+      is.empty(options) ||
+      (is.object(options) && (
+        is.undefined(options.method) ||
+        (options.method &&
+          CORS_SIMPLE_METHODS.includes(options.method.toUpperCase()))));
+  return {
+    headers: Object.assign(
+      { Accept: 'application/json; charset=utf-8' },
+      ! isCorsSimpleMethod ? {
+        'Content-Type': 'application/json',
```

Comment:
Have you checked if this could be removed completely, just in case we could simplify this a bit?